salesforce/10.15/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -3153,6 +3156,7 @@ The *Query* operation is now deprecated. Use the <<queryXmlStream, *Query Xml St
 | Configuration | String | Name of the configuration to use. | | x
 | Salesforce Query a| String |  Query string that specifies the object to query, the fields to return, and any conditions for including a specific object in the query. For more information, see Salesforce Object Query Language (SOQL). |  | x
 | Parameters a| Object |  Map with values for placeholders in *Salesforce Query*. |  |
+| Escape characters for parameters value a| Boolean | Escapes special characters for the parameters' value. | `false` |
```

Comment:
Can you clarify whether "parameters" refers to all field values or to the value of the "Parameters field"?